neofetch

```diff
@@ -2150,9 +2151,17 @@ get_packages() {
             has bulge      && tot bulge list
             has pm         && tot-safe pm list packages
             has evox       && tot cat /var/evox/packages/DB
-            has squirrel   && tot ls /var/packages
+            has squirrel   && tot ls -w 1 /var/packages
             has anise      && tot anise s --installed
             has am         && pac "$(am -f --less)"
+            has bonsai     && tot bonsai list
```

Comment:
I can't find a package manager named bonsai... searching for "bonsai linux package manager" doesn't give any meaningful results, besides from an [abandoned repo](https://github.com/mitchweaver/bonsai) (which uses the command bs instead of `bonsai`) and [another repo that only has two stars](https://github.com/StefanoBelli/bonsai). Which bonsai did you want to add support for?